srv/salt/ceph/mds/files/keyring.j2

```diff
@@ -1,6 +1,7 @@
 [{{ client }}]
         key = {{ secret }}
-        caps mds = "allow"
-        caps mon = "allow profile mds"
-        caps osd = "allow rwx"
+        caps mds = "allow *"                         
```

Comment:
Mh, we had the "allow *" caps in [SES4](https://github.com/SUSE/DeepSea/blob/SES4/srv/salt/ceph/mds/files/keyring.j2), which will not be refreshed if you upgrade to SES5.

```
ceph auth get mds.<mdsnode>:
        caps mds = "allow *"
        caps mgr = "allow profile mds"
        caps mon = "allow profile mds"
        caps osd = "allow rwx"
```
If you do a `ceph auth add mds.data1 -i /srv/salt/ceph/mds/cache/data1.keyring` if will fail with:

` Error EINVAL: entity mds.data1 exists but cap mds does not match`

Do we have to remove/reauth in order to get it right?